openapi.yaml

```diff
@@ -7856,9 +7854,14 @@ components:
         account: 1
       properties:
         for:
-          description: List of repository groups for which to calculate the metrics.
+          description: |
+            List of repository groups for which to calculate the metrics.
+            Use `null` as repository group to refer to every repository.
           items:
-            $ref: '#/components/schemas/RepositorySet'
+            items:
```

Comment:
We should leave the reference to RepositorySet and use my hack on adding a description that I detailed in private-api-spec

Reply:
I put RepositorySet reference back, but in the second element of anyOf I need anyway to replicate RepositoryID